feature/gribi/otg_tests/backup_nhg_multiple_nh_test/backup_nhg_multiple_nh_test.go

```diff
@@ -198,7 +176,8 @@ func configureDUT(t *testing.T, dut *ondatra.DUTDevice) {
 	// create VRF "vrfA" and assign incoming port under it
 	i1 := &oc.Interface{Name: ygot.String(p1.Name())}
 	gnmi.Replace(t, dut, d.Interface(p1.Name()).Config(), configInterfaceDUT(i1, &dutPort1, dut))
-	configNetworkInstanceInterface(t, dut, vrf1, p1.Name(), uint32(0))
+	// create VRF "vrfA"
+	configNetworkInstance(t, dut, vrf1)
```

Comment:
Why has this been changed? It wasn't introduced as a part of the DCGate code.

That breaks the test.

cc @cfernanz